### DISCUSSION TITLE
Widget-y top bar (like gnome)

### DISCUSSION THREAD

**Discussion by MartyMcFlyInTheSky** [i3/i3#5851] [Q&A]
I was trying out dmenu and polybar and while they are nice I really liked the way gnome placed icons in a more widget-y way. So you could click on it and see additional informations or change some settings. Can this be accomplished by polybar / dmenu as well or do I need another top bar?

**Comment by okraits** [comment]
polybar and dmenu are separate projects and not linked to i3 in any way. I would suggest to have a look at polybar's documentation to see what is possible.

**Reply by MartyMcFlyInTheSky** [reply]
Thank you. I also thought about looking int eww, looks very promising
